event_handler/event_handler_test.py

```diff
@@ -100,10 +100,10 @@ def test_data_sent_to_pubsub(client):
     "event_handler.publish_to_pubsub", mock.MagicMock(return_value=True)
 )
 def test_verified_pagerduty_signature(client):
-    signature = "sha1=" + hmac.new(b"websubsecret", b"Hello", sha1).hexdigest()
+    signature = "v1=" + hmac.new(b"websubsecret", b"Hello", sha256).hexdigest()
     r = client.post(
         "/",
         data="Hello",
-        headers={"User-Agent": "PagerDuty-Webhook/V3.0", "X-Pagerduty-Signature": signature},
+        headers={"User-Agent": "PagerDuty-Webhook/V3.0", "X-PagerDuty-Signature": signature},
```

Comment:
Should this be changing?

Reply:
Yep. This corrects it to match the input we would receive from the real PagerDuty API.